.circleci/config.yml

```diff
@@ -24,21 +24,42 @@ jobs:
           command: ./gradlew --stacktrace lint checkstyle
       - android/save-gradle-cache
       - android/save-lint-results
-  Build:
-    executor: 
+  Test:
+    executor:
       name: android/default
       api-version: "27"
     steps:
       - checkout
       - copy-gradle-properties
       - android/restore-gradle-cache
       - run:
-          name: Build
-          command: ./gradlew --stacktrace assembleDebug assembleRelease
+          name: Test
+          command: ./gradlew --stacktrace test
+      - android/save-gradle-cache
+  Build and upload to S3:
+    executor:
+      name: android/default
+      api-version: "27"
+    steps:
+      - checkout
+      - copy-gradle-properties
+      - android/restore-gradle-cache
+      - android/publish-to-s3:
+          publish_gradle_task: :WordPressLoginFlow:publishLibraryToS3
       - android/save-gradle-cache
 
 workflows:
   WordPress-Login-Flow-Android:
     jobs:
       - Lint
-      - Build
+      - Test:
+          filters:
+            tags:
+              only: /.*/
+      - Build and upload to S3:
+          requires:
+            - Test
```

Comment:
Love how the upload requires the tests 😄

Reply:
Well.. you'd, you made that happen in the utils repo 🙈